enrollment-server-onboarding/pom.xml

```diff
@@ -164,6 +160,13 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
+                    <attachClasses>true</attachClasses>
```

Comment:
We do not use this option in other PowerAuth pom.xml files, is it necessary?

Reply:
🤔 Interesting. Where do you use the war artifact as a dependency? This is according the documentation https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#attachClasses

It creates:
- `enrollment-server-onboarding-1.3.0.war`
- `enrollment-server-onboarding-1.3.0-classes.jar`

Which is used later on in 
```xml
<dependency>
    <groupId>com.wultra.security</groupId>
    <artifactId>enrollment-server-onboarding</artifactId>
    <version>${project.version}</version>
    <classifier>classes</classifier>
</dependency>
```

Maven does not build it without that but it is fine for IDE. Don't you have jar plugin even for war packaging?

Reply:
Ah, I see now. We always use jars for library dependencies, so we don't have an issue in other projects. I guess if the separation would be perfect, we would not need to use this option.